doc/md_docs.go

```diff
@@ -72,6 +72,17 @@ func GenMarkdownCustom(cmd *cobra.Command, w io.Writer, linkHandler func(string)
 		buf.WriteString(fmt.Sprintf("```\n%s\n```\n\n", cmd.UseLine()))
 	}
 
+	if len(cmd.Aliases) > 0 {
+		buf.WriteString("Aliases: ")
+		for _, alias := range cmd.Aliases {
+			buf.WriteString(fmt.Sprintf("`%s`", alias))
+			if alias != cmd.Aliases[len(cmd.Aliases)-1] {
+				buf.WriteString(", ")
+			}
+		}
```

Comment:
Your code is fine.

I would have used 

```go
"`"+strings.Join("`, `", cmd.Aliases)+"`"
```

But your code somehow respects the existing code. So why not. But then I would suggest you something like this

```suggestion
		for i, alias := range cmd.Aliases {
			if i > 0 {
				buf.WriteString(", ")
			}
			buf.WriteString(fmt.Sprintf("`%s`", alias))
		}
```

I way simpler to read and maintain

Reply:
I agree

Reply:
The reason I made it the way it was before was so that the last alias would not have a comma